BFF/v3/docs/content/fundamentals/apis/local.md

```diff
@@ -0,0 +1,194 @@
+---
+title: "Local APIs"
+weight: 10
+---
+
+A _Local API_ is an API that is located within the BFF host. Local APIs are implemented with the familiar ASP.NET abstractions of API controllers or minimal API endpoints. 
```

Comment:
Is "Minimal API" a noun or is "minimal" an adjective? I never know. I thought it was a noun.

Reply:
no idea.. I like it better as a noun